setup/action.yaml

```diff
@@ -1,36 +1,22 @@
 name: Setup Mondoo Action
 description: Installs Mondoo Client into a workflow
```

Comment:
We need a different description here since we're not actually installing

Reply:
I think we can now remove this action completely. We no longer need to setup/install mondoo.